docs/development/extensions-contrib/statsd.md

```diff
@@ -44,6 +44,10 @@ All the configuration parameters for the StatsD emitter are under `druid.emitter
 |`druid.emitter.statsd.includeHost`|Flag to include the hostname as part of the metric name.|no|false|
 |`druid.emitter.statsd.dimensionMapPath`|JSON file defining the StatsD type, and desired dimensions for every Druid metric|no|Default mapping provided. See below.|
 |`druid.emitter.statsd.blankHolder`|The blank character replacement as StatsD does not support path with blank character|no|"-"|
+|`druid.emitter.statsd.queueSize`|Maximum number of unprocessed messages in the message queue.|no|Default value of StatsD Client(4096)|
+|`druid.emitter.statsd.poolSize`|Network packet buffer pool size.|no|Default value of StatsD Client(512)|
+|`druid.emitter.statsd.processorWorkers`|The number of processor worker threads assembling buffers for submission.|no|Default value of StatsD Client(1)|
+|`druid.emitter.statsd.senderWorkers`|	The number of sender worker threads submitting buffers to the socket.|no|Default value of StatsD Client(1)|
```

Comment:
are these the same defaults that statsd client gets created without your patch?

Reply:
Yes @abhishekagarwal87 . I've tested on a cluster with these values. 
I've initialised them to always use the DEFAULTS of statsd client for these parameters 
https://github.com/apache/druid/pull/16283/files#diff-3792ee2083cfb5ebd2255b8f65c9c8d7bf92a82325e48bb9ae59d252a5d292a2R99